runtime/jcl/uma/se11_exports.xml

```diff
@@ -0,0 +1,25 @@
+<!-- 
+	Copyright (c) 2018, 2018 IBM Corp. and others
```

Comment:
Can you remove the unnecessary indentation of this copyright notice and trim trailing whitespace, please?